mne/viz/tests/test_3d.py

```diff
@@ -50,17 +51,18 @@ def test_plot_sparse_source_estimates():
     n_time = 5
     n_verts = sum(len(v) for v in vertices)
     stc_data = np.zeros((n_verts * n_time))
-    stc_data[(np.random.rand(20) * n_verts * n_time).astype(int)] = 1
+    stc_size = stc_data.size
+    stc_data[(np.random.rand(stc_size / 20) * stc_size).astype(int)] = \
+        np.random.rand(stc_data.size / 20)
```

Comment:
Need to add some random values to the data (1/20th or 5% here). Before, this just assigned 20 random indices to be 1, but this causes issues for `clim='auto'` because it can't produce 3 different color limits.